### PR TITLE
adds translation(ES) for advanced guides

### PR DESCRIPTION
**Pull request by anthony2025**
**What**: Adds translations in Spanish for the advanced guides

### CODE REVIEW

**Comment by kentcdodds** [comment]
Thanks so much for this @anthony2025! Hey @gillchristian, @Macxim, and @tylermcginnis, could  to give this a review? You're awesome! Thanks!

File: pages/advanced/content/es/refs.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  title: '`Ref Interior`',
+  subtitle: 'Como acceder el `ref` interno',
```
Comment (Macxim) [inline pages/advanced/content/es/refs.js]: `Cómo` lleva tilde.

File: pages/advanced/content/es/existing-css.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    ~~~callout {title: 'Recuerda esto', type: 'warning'}
+    Con CSS en JS, el objetivo es estilar componentes y reusarlos. Con esto en
+    mente, si necesitas estilar tu aplicación entera (usando selectores ~html~/~body~
+    o añadiendo estilos de reseteo), no lo harás con ~glamorous~ directamente.
+    En su lugar debes de usar CSS regular, o usar la API de ~glamor~ para inyectar
+    estos estilos de forma global.
+
+    Adicionalmente, en lugar de usar CSS global para estilar un elemento ~a~,
+    debes de crear un componente ~Link~ con todos los estilos que necesites,
```
Comment (Macxim) [inline pages/advanced/content/es/existing-css.js]: "debes crear" o "tienes que crear", pero no `debes de`.

File: pages/advanced/content/es/size.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    export default Root
+    ~~~
+
+    ~~~callout {title: 'Una Mejor Experiencia', type: 'success'}
+    Recomendamos usar ya sea
+    [~babel-plugin-module-resolver~](https://github.com/tleunen/babel-plugin-module-resolver)
+    o la opción [~resolve.alias~](https://webpack.js.org/configuration/resolve/#resolve-alias)
+    en la configuración de Webpack para que no tengas que importar desde el path completo.
+
+    Tienes las siguientes opciónes disponibles:
```
Comment (Macxim) [inline pages/advanced/content/es/size.js]: `opciones` sin tilde.

File: pages/advanced/content/es/size.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    1. ~glamorous/dist/glamorous.es.tiny.js~ - elige esta si usas Webpack@>=2 o Rollup
+    2. ~glamorous/dist/glamorous.cjs.tiny~ - elige esta si no estas transpilando ESModules
```
Comment (Macxim) [inline pages/advanced/content/es/size.js]: `si no estás transpilando` (tilde)

File: pages/advanced/content/es/size.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    3. ~glamorous/dist/glamorous.umd.tiny.js~ - elige esta si estas incluyendolo como un script (Tambien hay una version ~.min.js~)
```
Comment (Macxim) [inline pages/advanced/content/es/size.js]: - `estás incluyéndolo` (tildes)
- `también` (sin mayúscula y con tilde)

File: pages/advanced/content/es/ssr.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  title: 'Renderizado en el Servidor',
+  description: `
+    Ya que ambos ~glamor~ y ~react~ nos permiten renderizar en el servidor, ~glamorous~ tambien lo hace!
```
Comment (Macxim) [inline pages/advanced/content/es/ssr.js]: - `React` (mayúscula)
- `también` (tilde)

File: pages/advanced/content/es/ssr.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    Personalmente uso esta función en [mi website personal](https://github.com/kentcdodds/kentcdodds.com),
+    el cual es generado al momento que es compilado en el servidor. Aprende más sobre renderizado en el servidor:
+    [~react~ en el server](https://facebook.github.io/react/docs/react-dom-server.html) y
```
Comment (Macxim) [inline pages/advanced/content/es/ssr.js]: React (mayúscula)

**Comment by Macxim** [comment]
- I'm really not a fan of the Title Case which is very specific to English language.
- Sometimes you use `el prop` (masculino), sometimes `la prop` (feminino). We should stick to feminine, as it comes from `propiedad` (femenino)

Otherwise, it looks good 👍

**Comment by anthony2025** [comment]
Thanks for making it so easy to contribute, @kentcdodds. I just updated the pr with @Macxim suggestions.

**Comment by paulmolluzzo** [comment]
Thank you @anthony2025! @Macxim any last comments or can I merge this in?

File: pages/advanced/content/es/index.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  title: 'Guías avanzadas',
+  heading: `
+    Esta sección demuestra algunas ideas más avanzadas que puedes lograr con glamorous
```
Comment (Macxim) [inline pages/advanced/content/es/index.js]: `En esta sección encontrarás...`
o
`Esta sección expone...`

File: pages/advanced/content/es/ssr.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  title: 'Renderizado en el servidor',
+  description: `
+    Ya que ~glamor~ y ~React~ nos permiten renderizar en el servidor, ~glamorous~ tambien lo hace!
```
Comment (Macxim) [inline pages/advanced/content/es/ssr.js]: ¡~glamorous~ tambien lo hace!
(missing `¡`)

**Comment by Macxim** [comment]
I made two more comments.
When that's fixed, we're good! 👍

**Comment by anthony2025** [comment]
Thanks @Macxim, fixed.

**Comment by kentcdodds** [comment]
Thanks tons!

**Pull request merged**